src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$header-height: 60px;
+$header-logo-size: 40px;
+$text-hover-color: #00ACDC;
+$card-border-color: #F3F3F3;
+$grey-color-text: #616070;
+$dark-blue-color-text: #060B35;
+$title-hover-color: #34568b;
+$white-color-button: #fff;
+$blue-color-button: #00ACDC;
```

Comment:
Why did you repeat the variable with the same values? 
Need to fix it
```suggestion
$text-hover-color: #00ACDC;
$card-border-color: #F3F3F3;
$grey-color-text: #616070;
$dark-blue-color-text: #060B35;
$title-hover-color: #34568b;
$white-color-button: #fff;
```
<img width="230" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/209d1228-60d7-4ce9-ac25-c3f9b5375206">
<img width="253" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/95965d98-b9a4-4bcf-a898-a7c1731b4a25">